packages/v4/patternfly-docs/content/design-guidelines/components/wizard/wizard.md

```diff
@@ -122,6 +122,19 @@ Optional steps can be added to the wizard. Make steps optional when they do not
 * The review step is enabled once all required steps have been filled out.
 
 * Once required steps have been filled, a tertiary button would appear to the right of the next/back buttons. Once clicked, this button would jump users ahead to the final review step. 
+ 
+### Wizard with a panel
+
+<img src="./img/wizard-with-panel-closed.png" alt="Wizard with a closed panel"/>
```

Comment:
Would add a click hand on the button so it's obvious where the user would be clicking to open the drawer. It took a second for me to realize where it was